tests/test_diskann.cpp

```diff
@@ -12,17 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <fmt/format-inl.h>
+
 #include <catch2/catch_test_macros.hpp>
 #include <catch2/generators/catch_generators.hpp>
 #include <fstream>
 #include <iostream>
 #include <nlohmann/json.hpp>
 
+#include "test_index.h"
 #include "vsag/errors.h"
 #include "vsag/vsag.h"
 
 const std::string tmp_dir = "/tmp/";
 
+TEST_CASE_METHOD(fixtures::TestIndex, "diskann build test", "[ft][index][diskann]") {
+    vsag::Options::Instance().logger()->SetLevel(vsag::Logger::Level::kDEBUG);
```

Comment:
is there some way to set the logger with debug mode in the first line of each test case

Reply:
add in constructor of fixtures::TestIndex